.travis.yml

```diff
@@ -92,7 +96,8 @@ install:
   - export ANDROID_NDK_r15c="`pwd`/_ci/android-ndk-r15c"
 
 script:
-  - python3 ./jenkins.py
+  # Need to suppress verbose logging or Travis fails at 4 MB
+  - python3 ./jenkins.py | grep -E '^\-\-'
```

Comment:
Please note this change.

Reply:
Add `VERBOSE=0` environment varible for such toolchains:
* https://github.com/ingenue/hunter/blob/c8c5a33274e9c2ba5af0d55fb3805d68fb885d32/.travis.yml#L39-L40